pom.xml

```diff
@@ -231,7 +231,7 @@
                             -Xep:NullablePrimitive:ERROR
                             -Xep:NullOptional:ERROR
                             -XepExcludedPaths:.*/src/test/java/.*
-                            -XepOpt:NullAway:AnnotatedPackages=org.asynchttpclient.channel,org.asynchttpclient.config,org.asynchttpclient.cookie,org.asynchttpclient.exception,org.asynchttpclient.filter
+                            -XepOpt:NullAway:AnnotatedPackages=org.asynchttpclient.channel,org.asynchttpclient.config,org.asynchttpclient.cookie,org.asynchttpclient.exception,org.asynchttpclient.filter,org.asynchttpclient.oauth,org.asynchttpclient.proxy,org.asynchttpclient.resolver
```

Comment:
Let's break lines

Reply:
Unfortunately breaking lines will break everything - I guess NullAway expects this `XepOpt:NullAway:AnnotatedPackages` parameter to be a single line without any breaks.
We can invert this parameter, i.e. instead of `include only these packages` we can have `include all packages except the following`. And I intend to eventually do that. But this requires fixing classes in the root `org.asynchttpclient` package first

Reply:
Alright, let's keep it then.